### ISSUE TITLE
Save Visual Script as Text

### ISSUE THREAD

**Issue by mrjustaguy** [godotengine/godot-proposals#1766]
**Describe the project you are working on:**
Doesn't matter for me that much, but any Visual Script user could use this to be saved from the RSCC bug, and there are .tres and .res, .tscn and .scn so having .tvs and .vs makes sense.

**Describe the problem or limitation you are having in your project:**
Version Control & RSCC bug (until a fix can be found for that strange bug)

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
By storing the script as a Text file instead of a binary 1st off the RSCC bug wouldn't be destructive for the script even if it got a version of that bug somehow, because all the information would be in text format that would be readable by the editor as normal text and would save it without loss of data (unlike RSCC where it turns the whole binary in to just RSCC upon save when broken)
The 2nd use case is VC, as the data would then be representable in a VC Friendly format, readable to people.

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**
Basically VisualScript.data (in GDScript) saved and loaded in a text form as that is all the needed data for reconstruction of a Visual Script

**If this enhancement will not be used often, can it be worked around with a few lines of script?:**
Not in a few in GDScript.

**Is there a reason why this should be core and not an add-on in the asset library?:**
Addon would be way larger, and as .tres and .tscn are core, so should .tvs be and would have more of a positive impact on anyone using VS compared to anyone using .tres or .tscn as opposed to their binary counterparts (as those binary counterparts don't tend to break like VS does)

**Comment by Calinou** [comment]
Duplicate of https://github.com/godotengine/godot-proposals/issues/938.
